src/components/pages/Categories/Edit.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+  <app-category-edit
+    :category-id="categoryId"
+    :category-name="categoryName"
+    :loading="loading"
+    :error-message="errorMessage"
+    :done-message="doneMessage"
+    :access="access"
+    @edited-category-name="editedCategoryName"
+    @handle-submit="handleSubmit"
+  />
+</template>
+
+<script>
+import { CategoryEdit } from '@Components/molecules';
+
+export default {
+  components: {
+    appCategoryEdit: CategoryEdit,
+  },
+  data() {
+    return {
+      title: '',
+      content: '',
+    };
+  },
+  computed: {
+    categoryId() {
+      let { id } = this.$route.params;
+      id = parseInt(id, 10);
+      return id;
```

Comment:
[nits]
こちらは`let`ではなく`const`を使うように書き換えてみてください！

Reply:
constで試してみたのですが、idを再代入することができなくエラーがてしまいます。
この場合、どのような処理を行ったら良いでようか。


また、Articlesを参考にしてletで宣言を行った箇所なのですが、
Categoriesとで宣言方法を変える理由についても教えていただきたいです。

Reply:
@YumeYoshimura 
であればArticlesの書き方があまり良くないですね！
constを使える部分はなるべくconstを使用した方が良いので、

```suggestion
      const { id } = this.$route.params;
      return parseInt(id, 10);
```

でいけると思いますよ！

Reply:
いただいたコメントを元に宣言方法の修正を行いました。
733582c2909318e08cd5a5a67f7e6c79a6340aab